sbt-ash/src/main/scala/org/virtuslab/ash/DumpEventSchema.scala

```diff
@@ -26,14 +26,12 @@ object DumpEventSchema {
       writer <-
         outputFile.createIfNotExists(createParents = true).clear().newFileOutputStream().printWriter().autoClosed
     } writer.print(yamlPrinter.pretty(json))
+    outputFile
   }
 
-  def dumpEventSchemaTask(key: TaskKey[Unit]): Def.Setting[_] =
+  def dumpEventSchemaTask(key: TaskKey[JFile]): Def.Setting[_] =
     key := {
-      val c = (Compile / compile).value
-      c.readCompilations()
-      DumpEventSchema(
-        (key / dumpEventSchemaOutputFile).value.toScala,
-        (key / dumpEventSchemaCompilerPluginOutputFile).value.toScala)
+      (Compile / compile).value
+      DumpEventSchema((key / ashDumpEventSchemaOutputFile).value.toScala, target.value.toScala).toJava
```

Comment:
Out of curiosity, while `.toJava`/`JFile`? maybe worth documenting if that's sth tricky

Reply:
For easier understanding, renamed `JFile` to `File` and `File` to `SFile`